bin/create-embed-test-post.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Create embed test post.
+ *
+ * @package AMP
+ */
+
+if ( ! defined( 'WP_CLI' ) ) {
+	echo "Must be run in WP-CLI via: wp eval-file bin/create-embed-test-post.php\n";
+	exit( 1 );
+}
+
+$data_entries = array(
+	array(
+		'prepare' => 'amp_test_prepare_image_attachments',
+		'heading' => 'Media Gallery',
+		'content' => function( $data ) {
+			return sprintf( '[gallery ids="%s"]', implode( ',', $data['ids'] ) );
+		},
+	),
```

Comment:
This needs now to be fleshed out with all that can be thrown at content in a vanilla WordPress install.